tensorboard/webapp/metrics/views/card_renderer/scalar_card_test.ts

```diff
@@ -3631,6 +3631,103 @@ describe('scalar card', () => {
       expect(data[5].run).toEqual('3 c/run6');
       expect(data[6].run).toEqual('1 a/run7');
     }));
+
+    it('filters runs by hparam when enableHparamsInTimeSeries is true', fakeAsync(() => {
```

Comment:
It would be nice to refactor to share most of the setup code (perhaps in a beforeEach()) and make each test small and focused.